packages/lesswrong/lib/collections/moderationTemplates/views.ts

```diff
@@ -0,0 +1,24 @@
+import { ensureIndex } from '../../collectionUtils';
+import ModerationTemplates from './collection';
+
+declare global {
+  type ModerationTemplatesViewTerms = Omit<ViewTermsBase, 'view'> & ({
+    view: 'moderationTemplatesPage'|'moderationTemplatesQuickview',
```

Comment:
Is there any reason we need to split this into a union instead of just making `view` optional?

```suggestion
    view?: 'moderationTemplatesPage'|'moderationTemplatesQuickview',
```